Makefile

```diff
@@ -411,8 +411,10 @@ build_standalone_db_migrator: compile_standalone_db_migrator
 	make -f $(MAKEFILEPATH_PHOTON)/Makefile _build_standalone_db_migrator -e BASEIMAGETAG=$(BASEIMAGETAG) -e VERSIONTAG=$(VERSIONTAG)
 
 build_base_docker:
+	docker login -u $(REGISTRYUSER) -p $(REGISTRYPASSWORD)
```

Comment:
We should do the login only when the user and password are provided.

Reply:
right, I should check it.